configs/init/instill/seed/model_definitions.yaml

```diff
@@ -1,22 +1,131 @@
-- title: GitHub
-  id: github
-  name: "model-definitions/github"
-  uid: 909c3278-f7d1-461c-9352-87741bef11d3
-  documentationUrl: https://docs.instill.tech/integrations/models/http
-  icon: http.svg
-  model_spec: 
-      repo: 
-  model_instance_spec: 
-      repo: 
-      tag:        
-- title: Local
-  id: local
-  name: "model-definitions/local"
-  uid: 909c3278-f7d1-461c-9352-87741bef1abc
-  documentationUrl: https://docs.instill.tech/integrations/models/http
-  icon: http.svg
-  model_spec: 
-      repo: 
-  model_instance_spec: 
-      repo: 
-      tag: 
\ No newline at end of file
+- id: "github"
+  uid: "909c3278-f7d1-461c-9352-87741bef11d3"
+  title: "GitHub"
+  documentationUrl: "https://docs.instill.tech/integrations/models/github"
+  icon: "github.svg"
+  modelSpecification:
```

Comment:
@xiaofei-du should we name same as protobuf for consistency. In protobuf, it named [modelSpec](https://github.com/instill-ai/protobufs/blob/main/instill/model/v1alpha/model_definition.proto#L43) only

Reply:
refactored, thanks!